File-Formats/PDF/Convert-HTML-To-PDF/blazor.md

```diff
@@ -9,116 +9,124 @@ keywords: Assemblies
 ![Blazor_step3](htmlconversion_images/blazor_step3.png)
 
-Create a new cs file named ExportService under Data folder and include the following namespaces in the file.
-
-{% highlight c# tabtitle="C#" %}
+3. Create a new class file named ExportService under Data folder and include the following namespaces in the file.
 
-using Syncfusion.HtmlConverter;
-using Syncfusion.Pdf;
-using System.IO;
+   {% highlight c# tabtitle="C#" %}
 
-{% endhighlight %}
+   using Syncfusion.HtmlConverter;
+   using Syncfusion.Pdf;
+   using System.IO;
 
+   {% endhighlight %}
 
-Add the following method in the ExportService class
+4. Add the following code in the ExportService class.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-public MemoryStream CreatePdf()
-{
-    //Initialize HTML to PDF converter.
-    HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
-    //Convert URL to PDF.
-    PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
-    MemoryStream stream = new MemoryStream();
-    document.Save(stream);
-    return stream;
-}
+   public MemoryStream CreatePdf()
+   {
+      //Initialize HTML to PDF converter.
+      HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+      BlinkConverterSettings blinkConverterSettings = new BlinkConverterSettings();
+      //Set Blink viewport size.
+      blinkConverterSettings.ViewPortSize = new Syncfusion.Drawing.Size(1280, 0);
+      //Assign Blink converter settings to HTML converter.
+      htmlConverter.ConverterSettings = blinkConverterSettings;
+      //Convert URL to PDF document.
+      PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
+      //Create memory stream.
+      MemoryStream stream = new MemoryStream();
+      //Save the document to memory stream.
+      document.Save(stream);
+      return stream;
+   }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Register your service in the ConfigureServices method available in the Startup.cs class as follows.
+5. Register your service in the ConfigureServices method available in the Startup.cs class as follows.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-public void ConfigureServices(IServiceCollection services)
-{
-    services.AddRazorPages();
-    services.AddServerSideBlazor();
-    services.AddSingleton<WeatherForecastService>();
-    services.AddSingleton<ExportService>();
-}
+   public void ConfigureServices(IServiceCollection services)
+   {
+      services.AddRazorPages();
+      services.AddServerSideBlazor();
+      services.AddSingleton<WeatherForecastService>();
+      services.AddSingleton<ExportService>();
+   }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Inject ExportService in-to FetchData.razor using the following code.
+6. Inject ExportService in-to FetchData.razor using the following code.
```

Comment:
Inject ExportService into FetchData.razor using the following code.

Reply:
changed